addons/common/stringtable.xml

```diff
@@ -82,12 +82,8 @@
             <Korean>켬</Korean>
             <Polish>Włączone</Polish>
         </Key>
-        <Key ID="STR_ZEN_Common_Off">
-            <English>Off</English>
-            <German>Aus</German>
-            <Japanese>オフ</Japanese>
-            <Korean>끔</Korean>
-            <Polish>Wyłączone</Polish>
```

Comment:
Don't think this was supposed to be removed?

Reply:
I must have not pasted after I copied O_o
0488420